src/main/java/picard/sam/RevertSam.java

```diff
@@ -164,6 +167,10 @@
             "the program will exit with an Exception instead of exiting cleanly. Output BAM will still be valid.")
     public double MAX_DISCARD_FRACTION = 0.01;
 
+    @Argument(doc = "If SANITIZE=true discard duplicate records.  Duplicate records will have the same values for all field" +
+            "including tags.")
```

Comment:
I think you probably need to add a note here that "deduplicate" doesn't have anything do with the usual "is duplicate flag" definition of the word.  Maybe we can find a better word to use instead?